src/Console/ConsoleOutput.php

```diff
@@ -309,6 +309,7 @@ public function outputAs($type = null)
             return $this->_outputAs;
         }
         $this->_outputAs = $type;
+        return null;
```

Comment:
Return `null` or `void` is practically identical since you can't really differentiate between them.